pkgs/test/lib/src/runner/node/socket_channel.dart

```diff
@@ -3,7 +3,7 @@
 // BSD-style license that can be found in the LICENSE file.
 
 @JS()
-library _;
+library library_name;
```

Comment:
Why is this change necessary?

Reply:
To make a lint check happy. Now reverted together with the rest.